sdk/core/core-arm/package.json

```diff
@@ -6,7 +6,7 @@
     "url": "https://github.com/Azure/azure-sdk-for-js"
   },
   "sdk-type": "client",
-  "version": "1.0.0-preview.6",
+  "version": "1.0.0",
```

Comment:
`core-arm` should stay in preview

Reply:
@ramya-rao-a given that preview.6 is released - this should be preview.7 correct?

Reply:
Yes. It should be preview.7

Reply:
I've set it to preview.7 for now.